databricks/koalas/internal.py

```diff
@@ -340,7 +340,8 @@ def __init__(self, sdf: spark.DataFrame,
                  index_map: Optional[List[IndexMap]] = None,
                  scol: Optional[spark.Column] = None,
                  data_columns: Optional[List[str]] = None,
-                 column_index: Optional[List[Tuple[str]]] = None) -> None:
+                 column_index: Optional[List[Tuple[str]]] = None,
+                 column_names: Optional[List[str]] = None) -> None:
```

Comment:
How about `column_index_names`? `column_names` sounds ambiguous.

Reply:
I agree I'll change it to `column_index_names`.